Tests/Unit/EmogrifierTest.php

```diff
@@ -750,6 +756,43 @@ public function emogrifyFormatsCssDeclarations($cssDeclarationBlock, $expectedSt
     }
 
     /**
+     * Data provider for emogrifyInvalidDeclaration.
+     *
+     * @return array[]
```

Comment:
Like the other data providers it is an array of associative arrays whose values are string. Elsewhere string[][] is used.